Education.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace MarsQA_1.Pages
+{
+    class Education
+    {
+        private static IWebElement Education_ => Driver.driver.FindElement(By.XPath("//A[text()='Education']"));

```

Comment:
Naming is not so meaningful. Please revisit all your namings and remove the underscore as it's generally not standard practice

Reply:
OKay sure. I'll look into that on all the classes